### DISCUSSION TITLE
Search/Filter by GitHub Team

### DISCUSSION THREAD

**Discussion by anders-rapp-wcar** [community/community#9906] [Other Feature Feedback, Questions, & Ideas]
We have a large organization with thousands of repositories. Those repositories are managed by GitHub Teams. Currently there is no possibility to filter by teams which is very frustrating. Well, I suppose you can use the graphql but it's rather cumbersome when you just want to do a quick search. Why not have team:GITHUB_TEAM_SLUG as a parameter just as org:ORG??

**Comment by Avivhdr** [comment]
https://github.com/github/feedback/discussions/9023

**Comment by JensRoland** [comment]
This is a huge inconvenience and frankly puzzling how it still isn't possible.

The view I am mainly missing is open PRs by team, and as far as I can tell this query is not even possible through GraphQL.

> Note: It is possible to get "Open PRs created by members of team X" but not "Open PRS in repos owned by team X". The first one is nice, but meaningless since my developers often submit PRs to shared components which should be reviewed by the team responsible for that component; whereas repos my team owns often have PRs opened by other teams (security, dependabot, other teams needing a minor extension to an endpoint) and my team needs to address those. "Review requested from team" isn't a good enough query either since not all repos are configured to automatically request reviews from the owning team.

For our org, I had to build a CLI tool that calls several API endpoints to fetch and list the PRs for a team, and while it works, it is slow and requires manual polling. As a result of this one blind spot, PRs sit idle/unnoticed far longer (slowing down dev velocity) with GitHub than when using other tools.

It's odd since the GitHub dev tooling is superior in most other ways, but this is just plain missing batteries.

**Comment by shonfeder** [comment]
2018: https://github.com/dear-github/dear-github/issues/306
